### ISSUE TITLE
Laravel 5.8 support?

### ISSUE THREAD

**Issue by 4n70w4** [brexis/laravel-workflow#48]
`composer require brexis/laravel-workflow`

```
Using version ^1.2 for brexis/laravel-workflow
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install brexis/laravel-workflow 1.2.3
    - Conclusion: don't install brexis/laravel-workflow 1.2.2
    - Conclusion: don't install brexis/laravel-workflow 1.2.1
    - Conclusion: remove laravel/framework v5.8.13
    - Installation request for brexis/laravel-workflow ^1.2 -> satisfiable by brexis/laravel-workflow[1.2.0, 1.2.1, 1.2.2, 1.2.3].
    - Conclusion: don't install laravel/framework v5.8.13
    - brexis/laravel-workflow 1.2.0 requires illuminate/console 5.3.* || 5.4.* || 5.5.* || 5.6.* -> satisfiable by laravel/framework[5.3.x-dev, 5.4.x-dev, 5.5.x-dev, 5.6.x-dev], illuminate/console[5.3.x-dev, 5.4.x-dev, 5.5.x-dev, 5.6.x-dev, v5.3.0, v5.3.16, v5.3.23, v5.3.4, v5.4.0, v5.4.13, v5.4.17, v5.4.19, v5.4.27, v5.4.36, v5.4.9, v5.5.0, v5.5.16, v5.5.17, v5.5.2, v5.5.28, v5.5.33, v5.5.34, v5.5.35, v5.5.36, v5.5.37, v5.5.39, v5.5.40, v5.5.41, v5.5.43, v5.5.44, v5.6.0, v5.6.1, v5.6.10, v5.6.11, v5.6.12, v5.6.13, v5.6.14, v5.6.15, v5.6.16, v5.6.17, v5.6.19, v5.6.2, v5.6.20, v5.6.21, v5.6.22, v5.6.23, v5.6.24, v5.6.26, v5.6.27, v5.6.28, v5.6.29, v5.6.3, v5.6.30, v5.6.31, v5.6.32, v5.6.33, v5.6.34, v5.6.35, v5.6.36, v5.6.37, v5.6.38, v5.6.39, v5.6.4, v5.6.5, v5.6.6, v5.6.7, v5.6.8, v5.6.9].
    - Can only install one of: laravel/framework[5.3.x-dev, v5.8.13].
    - Can only install one of: laravel/framework[5.4.x-dev, v5.8.13].
    - Can only install one of: laravel/framework[5.5.x-dev, v5.8.13].
    - Can only install one of: laravel/framework[5.6.x-dev, v5.8.13].
    - don't install illuminate/console 5.5.x-dev|don't install laravel/framework v5.8.13
    - don't install illuminate/console 5.6.x-dev|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.0|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.16|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.17|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.2|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.28|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.33|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.34|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.35|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.36|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.37|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.39|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.40|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.41|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.43|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.5.44|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.0|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.1|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.10|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.11|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.12|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.13|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.14|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.15|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.16|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.17|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.19|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.2|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.20|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.21|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.22|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.23|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.24|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.26|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.27|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.28|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.29|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.3|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.30|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.31|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.32|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.33|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.34|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.35|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.36|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.37|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.38|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.39|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.4|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.5|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.6|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.7|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.8|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.6.9|don't install laravel/framework v5.8.13
    - don't install illuminate/console 5.3.x-dev|don't install laravel/framework v5.8.13
    - don't install illuminate/console 5.4.x-dev|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.3.0|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.3.16|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.3.23|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.3.4|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.4.0|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.4.13|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.4.17|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.4.19|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.4.27|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.4.36|don't install laravel/framework v5.8.13
    - don't install illuminate/console v5.4.9|don't install laravel/framework v5.8.13
    - Installation request for laravel/framework (locked at v5.8.13, required as 5.8.*) -> satisfiable by laravel/framework[v5.8.13].


Installation failed, reverting ./composer.json to its original content.
```

**Comment by 4n70w4** [comment]
https://github.com/brexis/laravel-workflow/pull/47

**Comment by 4n70w4** [comment]
workaround: https://packagist.org/packages/symfony-rot-ebal/laravel-workflow

**Comment by michaeln-recommerce** [comment]
Have same issue.

**Comment by zayaslui** [comment]
example in proyect laravel with config.

**Comment by NettSite** [comment]
> workaround: https://packagist.org/packages/symfony-rot-ebal/laravel-workflow

What is the workaround?

**Comment by NettSite** [comment]
> example in proyect laravel with config.

Any more detail?

**Comment by NettSite** [comment]
> > workaround: https://packagist.org/packages/symfony-rot-ebal/laravel-workflow
> 
> What is the workaround?

For other stupid people like me, the workaround is to install using
 `composer require symfony-rot-ebal/laravel-workflow`

**Comment by zerodahero** [comment]
This package seems to be either unmaintained or very slowly maintained (see #51 ). Many people (including myself) have forked this and are opting to maintain their own version rather than wait for changes in this one. You can [search packagist](https://packagist.org/?query=laravel-workflow) (the main Composer repo) for 'laravel-workflow' to find alternatives that you can use in place of this one.

**Comment by pavan-cs** [comment]
How did you resolve this issue?? @4n70w4 @NettSite

**Comment by NettSite** [comment]
@pavan-cs I installed the package as follows:
`composer require symfony-rot-ebal/laravel-workflow`

**Comment by pavan-cs** [comment]
> @pavan-cs I installed the package as follows:
> `composer require symfony-rot-ebal/laravel-workflow`

one more stupid here :)
thanks.

**Comment by brexis** [comment]
Fixed on the last release 1.3.0.
